### DISCUSSION TITLE
Internal player not working, VLC no problem.

### DISCUSSION THREAD

**Discussion by pixel24** [4gray/iptvnator#380] [General]
Hi all,

Thanks for the software! I've been searching for an IPTV client for Linux that also supports EPG for a while :-)

I installed version 0.15.1 as a DEB package (Linux Mint 21.3). I entered my M3U and XMLTV URLs, and all channels are showing up. However, playback does not start; the loading circle spins indefinitely, and a message at the top indicates there is a network problem, which is not the case.

When I change the player to VLC in the IPTVnater settings, everything works fine without any issues. None of the other player options available work either and exhibit the same behavior.

This is not critical for me; I can use it this way :-)

Is there a way to embed the VLC player into the IPTVnater window?

with best

**Comment by sevbest** [comment]
I having  a similar experience. The internal video player does not function with stalker server (I tried against 10's) but work fine with VLC. The internal player works fine for Xtream server however...
